src/gb/GB.cpp

```diff
@@ -3443,8 +3443,8 @@ bool gbReadGSASnapshot(const char* fileName)
         return false;
     }
     fseek(file, 0x13, SEEK_SET);
-    size_t read = 0;
-    int toRead = 0;
+    //size_t read = 0;
+    //int toRead = 0;
```

Comment:
There is something weird here. This var `toRead` is never used but it is update on a `switch`. Is this a TODO somewhere?

Reply:
That's fine then, the original author thought he needed it but didn't. Leaving it as a comment is fine.